### PR TITLE
Opt-out doc for Mobile-SDK's

### PR DESCRIPTION
**Pull request by 1abhishekpandey**
## Description of the change

> Description here

## Type of change
- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Related issues

> Fix [#1]() 

### Development

- [ ] Lint rules pass locally
- [ ] The code changed/added as part of this pull request has been covered with tests
- [ ] All tests related to the changed code pass in development

### Code review 

- [ ]  This pull request has a descriptive title and information useful to a reviewer. There may be a screenshot or screencast attached
- [ ] "Ready for review" label attached to the PR and reviewers mentioned in a comment
- [ ] Changes have been reviewed by at least one other engineer
- [ ] Issue from task tracker has a link to this pull request

### CODE REVIEW

File: stream-sources/rudderstack-sdk-integration-guides/rudderstack-ios-sdk.md [hunk 1]
```diff
@@ -131,6 +131,28 @@ You can disable these events using the`withTrackLifecycleEvents` method of `RSCo
 
 RudderStack supports all the major API calls across all iOS devices via the SDK. These include the `track`, `identify`, and `screen` calls.
 
+## Opt-out
+
+You can provide the ability to the user (for example, EU user) to opt-out from tracking the activity, using the below `optOut` API snippet. Opt-out could be set to `YES` or `NO` to disable or enable tracking user activity. This flag persists across device reboots.
```
Comment (pallabmaiti) [inline stream-sources/rudderstack-sdk-integration-guides/rudderstack-ios-sdk.md:136]: @1abhishekpandey I changed this section slightly above. Can you change accordingly?

Comment (1abhishekpandey) [reply stream-sources/rudderstack-sdk-integration-guides/rudderstack-ios-sdk.md]: Done, thanks.

File: stream-sources/rudderstack-sdk-integration-guides/rudderstack-android-sdk/README.md [hunk 2]
```diff
@@ -136,6 +136,32 @@ We automatically track the following optional events:
 You can disable these events using the `withTrackLifecycleEvents` method and passing `false`. But it is highly recommended to keep them enabled.
 {% endhint %}
 
+## Enabling/Disabling User Tracking via the optOut API (GDPR Support)
+
+RudderStack gives you (e.g., an EU user) the ability to opt out of tracking any user activity by leveraging its `optOut` API.
```
Comment (ameypv-rudder) [inline stream-sources/rudderstack-sdk-integration-guides/rudderstack-android-sdk/README.md:141]: I'm assuming you can opt out of tracking any user activity until they give their consent. Should we be more specific here if that's the case?

File: stream-sources/rudderstack-sdk-integration-guides/rudderstack-android-sdk/README.md [hunk 3]
```diff
@@ -136,6 +136,32 @@ We automatically track the following optional events:
 You can disable these events using the `withTrackLifecycleEvents` method and passing `false`. But it is highly recommended to keep them enabled.
 {% endhint %}
 
+## Enabling/Disabling User Tracking via the optOut API (GDPR Support)
+
+RudderStack gives the users (e.g., an EU user) the ability to opt out of tracking any user activity by leveraging its `optOut` API. 
+
```
Comment (ameypv-rudder) [inline stream-sources/rudderstack-sdk-integration-guides/rudderstack-android-sdk/README.md:142]: I'm assuming you can opt out of tracking any user activity until the user gives their consent. Should we be more specific here if that's the case?

File: stream-sources/rudderstack-sdk-integration-guides/rudderstack-android-sdk/README.md [hunk 4]
```diff
@@ -136,6 +136,32 @@ We automatically track the following optional events:
...
+An example of how you can use the `optOut` API is shown below:
+
+{% tabs %}
+{% tab title="Kotlin" %}
+```kotlin
+rudderClient.optOut(true)
+```
+{% endtab %}
+
```
Comment (ameypv-rudder) [inline stream-sources/rudderstack-sdk-integration-guides/rudderstack-android-sdk/README.md:153]: if the above comment is true, should we also mention the case that once the user gives the consent, we can enable user tracking again by passing the `false` boolean to the optOut API?

File: stream-sources/rudderstack-sdk-integration-guides/rudderstack-android-sdk/README.md [hunk 5]
```diff
@@ -136,6 +136,32 @@ We automatically track the following optional events:
 You can disable these events using the `withTrackLifecycleEvents` method and passing `false`. But it is highly recommended to keep them enabled.
 {% endhint %}
 
+## Enabling/Disabling User Tracking via the optOut API (GDPR Support)
```
Comment (ameypv-rudder) [inline stream-sources/rudderstack-sdk-integration-guides/rudderstack-android-sdk/README.md:139]: I changed the heading for better clarity. LMK if it looks okay.

File: stream-sources/rudderstack-sdk-integration-guides/rudderstack-android-sdk/README.md (hunk 3 above)
Comment (1abhishekpandey) [reply stream-sources/rudderstack-sdk-integration-guides/rudderstack-android-sdk/README.md:142]: Yes, it'll be better if we be more specific.

**Review by ameypv-rudder — APPROVED** [review]

**Review by pallabmaiti — APPROVED** [review]

**Review by arnab-p — APPROVED** [review]

**Pull request merged**